docs/docs/references/users/README.md

```diff
@@ -214,3 +214,69 @@ Updating a user's _custom data_ will overwrite (not merge) its original content
 :::
 
 ### Identities
+
+_Identities_ contain the user info retrieved from social sign-in
+(a.k.a. sign-in with a social connector).
+
+The user info varies by provider,
+but it will typically include the following:
+
+- Name of this provider, such as lowercase "facebook", "google", or "wechat"
```

Comment:
`Name` -> `Target` to align with the term in connector. Also, add a link to the "connector reference" page so that developers can explore more details.

Reply:
Updated.